android/build.gradle

```diff
@@ -1,10 +1,14 @@
 buildscript {
   repositories {
     jcenter()
+    maven {
+      url 'https://maven.google.com/'
+      name 'Google'
+    }
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:1.1.3'
+    classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
I see gradle has been updated and hence it requires the `maven` repo and also the `jcenter` repos.

Can you test if this is working as expected when trying to compile with NiceDay on another machine?

Reply:
I'm going to check this first on another machine and will update here whether it's working or not.

Reply:
@coldpinguin just helped me checked on his machine and it's compiled and built.